src/modules/UI/components/ProgressBar/ProgressBar.ui.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import React, { Component } from 'react'
+import { View, Animated, Easing } from 'react-native'
+import style from './styles.js'
+
+type Props = {
+  progress: number
+}
+
+export default class ProgressBar extends Component<Props> {
+  animation: Animated.Value
+
+  componentWillMount () {
+    this.animation = new Animated.Value(this.props.progress)
+  }
+
+  componentDidUpdate (prevProps: Props) {
```

Comment:
If you make this a "PureComponent" then it will always check if the props have changed before calling componentDidUpdate or render()

Change to:

`export default class ProgressBar extends PureComponent<Props> {`